### ISSUE TITLE
QUESTION: Intel 1st Gen (Arrandale)

### ISSUE THREAD

**Issue by renan4nandes** [sebanc/brunch#1350]
Is there any support for newer versions of ChromeOS to Intel 1st Gen processors (like mine, a Intel Core i3 M370)?

EDIT: i just want to be sure about it, i already read README.md

EDIT 2: version 81 of ChromeOS in https://cros.tech is gone

**Comment by cambarevalo** [comment]
Hi there,
I found the rammus 81 recovery image download link in the discord, credits to the mod Wisteria#1539
https://dl.google.com/dl/edgedl/chromeos/recovery/chromeos_12871.102.0_samus_recovery_stable-channel_mp-v3.bin.zip
Hope this helps!

**Comment by tejasraman** [comment]
No, Google completely removed support on r81. You could try Cloudready, but YMMV

**Comment by tejasraman** [comment]
Use ChromeOS flex

**Comment by renan4nandes** [comment]
they have support for arrandale?

**Comment by tejasraman** [comment]
Yes, chrome on flex has support for arrandale, but no android or Linux.
https://www.businessinsider.com/chrome-os-flex
You can ignore the supported devices list; chrome os flex uses a Linux kernel (albeit an older version, but your computer should work with it)

**Comment by tejasraman** [comment]
You can get it at chrome enterprise: https://chromeenterprise.google/os/chromeosflex/
